package.json

```diff
@@ -9,6 +9,9 @@
     "type": "vue-tsc --noEmit",
     "lint": "eslint --ext .js,.vue --ignore-path .gitignore --fix './{lib,tests}/**/*.{ts,vue}'",
     "lint:fail": "eslint --ext .js,.vue --ignore-path .gitignore './{lib,tests}/**/*.{ts,vue}'",
+    "story:dev": "histoire dev",
```

Comment:
Can we make this `story` (without `:dev`) instead? 👀

Reply:
Yes, it looks better!